packages/adapter-vercel/index.js

```diff
@@ -140,6 +140,10 @@ const plugin = function (defaults = {}) {
 						target: 'es2020', // TODO verify what the edge runtime supports
 						bundle: true,
 						platform: 'browser',
+						conditions: [
+							'edge-light',
+							'module' // Matching behavior of esbuild without custom conditions
```

Comment:
```suggestion
							// Vercel's Edge runtime key https://runtime-keys.proposal.wintercg.org/#edge-light
							'edge-light',
							// re-include these since they are included by default when no conditions are specified
							// https://esbuild.github.io/api/#conditions
							'module'
```